docs/setup/custom_config.md

```diff
@@ -3,5 +3,8 @@
 If you'd like to override configuration values, you can create for every environment in the *app/config* directory a custom configuration file:
 
     app/config/local_config_{environment}.yml
+If you want to create a new account, you will have to add a line called "secret_key" to your parameters.yml.
```

Comment:
add the value with default to '~' in the parameters.yml.dist then composer will add it automatically to the parameters file. It's only necessary to customize the value. (sry if I was unclear about this when chatting)